docs/fsharp-core-notes.md

```diff
@@ -1,126 +1,124 @@
-using an **explicit** reference, especially when creating libraries.
+|Minimum F# language version|Minimum FSharp.Core package version|
+|------------------------------|------------------------------|
+|F# 4.1|4.3.4|
+|F# 4.5|4.5.2|
+|F# 4.6|4.6.2|
+|F# 4.7|4.7.2|
+|F# 5.0|5.0.0|
 
-To select a particular FSharp.Core use `Update`:
+If you want to be compatible with much older projects using an F# 4.0 compiler or earlier, you can still do that but it's not recommended. People using those codebases should upgrade instead.
 
-    <PackageReference Update="FSharp.Core" Version="4.7.2" />
+### Do *not* bundle FSharp.Core directly with a library 
 
-In C# projects use:
+Do _not_ include a copy of FSharp.Core with your library or package, such in the `lib` folder of a package. If you do this, you will create havoc for users of your library.
 
-    <PackageReference Include="FSharp.Core" Version="4.7.2" />
+The decision about which `FSharp.Core` a library binds to is up to the application hosting of the library.
 
-If you make your FSharp.Core dependency explicit, you will have to explicitly upgrade your FSharp.Core reference in order to use
-new F# language or library features should those features depend on a particular minimal FSharp.Core version.
+## Guidance for everyone else
 
-### Libraries should target lower versions of FSharp.Core
+If you're not authoring packages for distribution, you have a lot less to worry about.
 
-F# ecosystem libraries should generally target the *earliest, most portable* profile of FSharp.Core feasible, within reason.
+If you are distributing library code across a private organization as if it were a NuGet package, please see the above guidance, as it likely still applies. Otherwise, the below guidance applies.
 
-If your library is part of an ecosystem, it can be helpful to target the _earliest, most widespread language version_ 
-and the _earliest_ and _most supported_ profiles of the .NET Framework feasible.
+### Application authors don't have to explicitly reference FSharp.Core
 
-The version you choose should be based on the minimum F# language version you want to support. The minimum FSharp.Core version for each language version is listed below:
+In general, applications can always just use the latest FSharp.Core bundled in the SDK they are built with.
 
-|Minimum F# language version|Minimum FSharp.Core version|
-|------------------------------|------------------------------|
-|F# 4.1|4.3.4|
-|F# 4.5|4.5.2|
-|F# 4.6|4.6.2|
-|F# 4.7|4.7.2|
-|F# 5.0|5.0.0|
+### C# projects referencing F# projects may need to pin FSharp.Core
+
+You can reference an F# project just fine without needing to be explicit about an FSharp.Core reference when using non-legacy projects. References flow transitively, so even if you need to use types directly from FSharp.Core (which you probably shouldn't do anyways) it will pick up the right types from the right assembly.
```

Comment:
Drop "when using non-legacy projects" - all mention of legacy can be elsewhere

Reply:
"just fine" sounds a bit odd

Reply:
This is actually something worth having for C# projects. The large majority of C# projects in the wild are legacy projects, and the project system that powers them does not know how to handle transitive references. This is unlike the F# world, where the majority of users have upgraded to SDK-style.